src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
+
+  if (days < 3) {
+    return rent * days;
+  }
+
+  if (days >= 3 && days < 7) {
+    return rent * days - 20;
+  }
+
+  if (days >= 7) {
+    return rent * days - 50;
+  }
```

Comment:
1. move all magic numbers into constants
2. rent * days is duplicated 3 times, move it into variable` const price = days * 40;`